feature-libs/product-multi-dimensional/selector/components/product-multi-dimensional-selector-components.module.ts

```diff
@@ -5,9 +5,9 @@
  */
 
 import { NgModule } from '@angular/core';
-import { ProductMultiDimensionalSelectorModule } from './selector/product-multi-dimensional-selector.module';
+import { ProductMultiDimensionalSelectorComponentModule } from './selector/product-multi-dimensional-selector-component.module';
```

Comment:
`minor:` Based on similar places in the app, I think you don't need to mention `Component` in the name of the class and the file. `ProductMultiDimensionalSelectorModule` would be good enough.

Reply:
As discussed offline, due to naming conflict with the root module of this sub-feature, let's keep `ProductMultiDimensionalSelectorComponentModule`